docs/API.md

```diff
@@ -769,7 +777,7 @@ Added in supervisor version v7.0.0. Support for passing `serviceName` instead of
 `imageId` added in v8.2.2.
 
 Use this endpoint to restart a service in the application with application id
-passed in with the url.
+passed in with the url. Note that this removes and recreates the service container.
```

Comment:
It is, I was wrong in the last product call since I was looking at `balena ps` output, which showed that container ID had remained the same, whereas Supervisor had actually removed & re-created the individual container when it was restarted. [Removing a container is the default](https://github.com/balena-os/balena-supervisor/blob/master/src/compose/service-manager.ts#L515) when executing a kill step. I've updated the JF brainstorm topic & the issues where I mentioned this to convey the right information.